src/main/webapp/app/tasks/task-list/task-list.component.ts

```diff
@@ -17,6 +15,7 @@ export class TaskListComponent {
   @Input() tasks: Task[];
 
   @Output() deleted: EventEmitter<Task> = new EventEmitter();
+  searchTerms = '';
```

Comment:
You don't even need this in the component if you are just passing it along in the template.